src/talos/GeneratePED.py

```diff
@@ -22,10 +22,10 @@
 HPO_RE = re.compile(r'HP:[0-9]+')
 PARTICIPANT_QUERY = gql(
     """
-query MyQuery($project: String!) {
+query MyQuery($project: String!, $sequencing_type: String!) {
   project(name: $project) {
     pedigree
-    sequencingGroups {
+    sequencingGroups(type:  {eq: $sequencing_type}) {
```

Comment:
You could consider adding a filter for sequencing group technology as well, like `technology: {eq: "short-read"}`.

I'm just noting that below, this query is parsed and each sequencing group is keyed by the participant external ID. In some cases, we can have participants with multiple sequencing groups of the same sequencing type but  with differing technologies (or platforms). 

If you were parsing a project that had a participant with both a "long-read" and "short-read" sequencing group of type = "genome", then you will lose one of these sequencing groups as you construct the `ext_to_int` dict.

Reply:
Nice, hadn't really considered that angle